### PR TITLE
Add bg ref page

### LINKED ISSUE #353: Add description to bg reference page
bg is an important command for job control. An updated reference page could include links to related commands fg and jobs.

I will work on this.

### PR DESCRIPTION
**Pull request by evandhof**
# Advanced HW 5 submission
Fill out the ref page for the bg command

Closes #353 

Name: Evan Hoffman
Uniqname: evandhof

### CODE REVIEW

File: _commands/basics/bg.md [hunk 1]
```diff
@@ -3,22 +3,35 @@
...
 
 ### Useful Options / Examples
 
-#### Example command
+`bg` can be passed job numbers as well as job names
 
-##### Break it down
+~~~ bash
+$ jobs
+[1]+ Stopped    sleep 100
```
Comment (cameron-gagnon) [inline _commands/basics/bg.md]: This `sleep 100` seems to just appear. Try including the `sleep` command as well in the output that you show here.

File: _commands/basics/bg.md [hunk 2]
```diff
@@ -3,22 +3,35 @@
...
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+`bg` can be passed job numbers as well as job names
 
-##### Break it down
+~~~ bash
+$ jobs
```
Comment (cameron-gagnon) [inline _commands/basics/bg.md]: Could also be useful to see that `jobs` produces no output when it's run the first time.

File: _commands/basics/bg.md [hunk 3]
```diff
@@ -3,22 +3,35 @@
...
+$ bg 2
+$ jobs
+[1]+ Stopped    sleep 100
+[2]+ Running    gedit file.txt
+$
+~~~
 
-#### Example command
+### Related Commands
+[fg](../commands/fg), [jobs](../commands/jobs)
```
Comment (cameron-gagnon) [inline _commands/basics/bg.md]: `kill` is also often used together with these commands.

**Comment by evandhof** [comment]
@cameron-gagnon added suggested changes, any more suggestions?

File: _commands/basics/bg.md [hunk 4]
```diff
@@ -23,15 +23,24 @@ $
...
 [2]+ Stopped    gedit file.txt
 $ bg 2
 $ jobs
 [1]+ Stopped    sleep 100
 [2]+ Running    gedit file.txt
+$ bg 'sleep 100'
+$ jobs
+[1]+ Running    sleep 100
+[2]+ Running    gedit file.txt
 $
```
Comment (cameron-gagnon) [inline _commands/basics/bg.md]: Could also be useful to show how to background multiple jobs at once.

**Comment by cameron-gagnon** [comment]
One last request!

File: _commands/basics/bg.md (hunk 4 above)
Comment (evandhof) [reply _commands/basics/bg.md]: @cameron-gagnon so added

**Pull request merged**